packages/embark-contracts-manager/src/index.js

```diff
@@ -135,65 +144,53 @@ class ContractsManager {
 
             if (error) {
               self.events.emit('contracts:log', contractLog);
-              return res.status(500).send({result: error.message});
+              return res.status(200).send({result: error.message});
```

Comment:
I think we should actually return `500` here no?

Reply:
Yes, agreed, but Cockpit expects a 200, so this was to create feature parity. Something we should fix later.